modules/hub/hub.go

```diff
@@ -157,3 +157,10 @@ func GetGithubIssues(token string, repo *repository.RepositoryStruct) ([]github.
 
 	return opneIssues, closedIssues, nil
 }
+
+func IsPullRequest(issue *github.Issue) bool {
```

Comment:
__[golint]__
* [golint] exported function IsPullRequest should have comment or be unexported